.github/workflows/buid-documentation.yml

```diff
@@ -21,14 +21,15 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Compile Coq-Doc
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+        #if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         uses: coq-community/docker-coq-action@v1
         with:
           custom_image: ${{ matrix.image }}
           before_script: |
             startGroup "Create Makefile.coq and add permission"
-              make || true
-              sudo chown -R coq:coq .
+              sudo chown -R coq:coq Makefile
```

Comment:
It doesn't work if you do `sudi chown -R coq:coq .` before `make Makefile.coq`?

Reply:
Thanks, that was the problem.